plugins/inputs/jolokia/jolokia_test.go

```diff
@@ -175,7 +175,7 @@ func TestHttpJsonOn404(t *testing.T) {
 
 	var acc testutil.Accumulator
 	acc.SetDebug(true)
-	err := jolokia.Gather(&acc)
+	err := acc.GatherError(jolokia.Gather)
 
 	assert.Nil(t, err)
```

Comment:
This test is failing because of a similar situation to the cassandra one. The code was previously printing an error, but is now being passed to `AddError()`. The problem here is that this test doesn't make much sense. It generates a stub that returns valid JSON, but with a http 404 status code. Jolokia will not do this, thus the test seems bogus.

Given that the cassandra code seems to be based off the jolokia code, I'm betting the cassandra test that I previously commented on is bogus too.